Apps/phcuckoo/cuckoo_connector.py

```diff
@@ -226,6 +226,7 @@ def _make_rest_call(self, endpoint, action_result, headers=None, params=None, da
         # Create a URL to connect to
         url = "{0}{1}".format(self._base_url, endpoint)
 
+        print("auth", self._auth, "headers", headers, "data", data, "json", json, "files", files)
```

Comment:
Do any of these expose sensitive pieces of data?

Reply:
Fixed